pkg/util/proxy/proxy.go

```diff
@@ -122,6 +123,7 @@ func NewThrottledUpgradeAwareProxyHandler(location *url.URL, transport http.Roun
 	return proxy.NewUpgradeAwareHandler(location, transport, wrapTransport, upgradeRequired, proxy.NewErrorResponder(responder))
 }
 
+// GetTlsConfigForCluster returns a tls config for the cluster.
```

Comment:
```suggestion
// GetTLSConfigForCluster returns a tls config for the cluster.
func GetTLSConfigForCluster(ctx context.Context, cluster *clusterapis.Cluster, secretGetter SecretGetterFunc) (*tls.Config, error) {
```
Refer to 4496, iit may be better to change it this way to avoid repeated modifications in the same place.

Reply:
fixed